File: Seccion 11 - Funciones en C++/Seccion_11_Dart/02_plantillas.dart

void mostrar_ABS(dynamic numero) {
  if (numero < 0) {
    numero = numero * -1;
  }

  print('El valor absoluto del número es: $numero');
}

void main() {

  int num1 = -4;
  double num2 = -56.25;

  mostrar_ABS(num1);
  mostrar_ABS(num2);

}

File: Seccion 11 - Funciones en C++/Seccion_11_Python/02_plantillas.py

def mostrar_ABS(numero):
    if numero < 0:
        numero = numero * -1
    
    print('El valor absoluto del número es: {}'.format(numero))

num1 = -4
num2 = -56.25

mostrar_ABS(num1)
mostrar_ABS(num2)